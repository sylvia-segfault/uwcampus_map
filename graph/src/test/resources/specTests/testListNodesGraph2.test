# Create an empty graph
CreateGraph graph2

# Add a pair of nodes
AddNode graph2 node1
AddNode graph2 node2

# Returns a list of vertices in graph2
ListNodes graph2