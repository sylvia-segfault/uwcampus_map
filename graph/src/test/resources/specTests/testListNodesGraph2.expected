# Create an empty graph
created graph graph2

# Add a pair of nodes
added node node1 to graph2
added node node2 to graph2

# Returns a list of vertices in graph2
graph2 contains: node1 node2